pkg/hub/submarineragent/manifests/operator/submariner.io-submariners-cr.yaml

```diff
@@ -42,3 +42,4 @@ spec:
 {{- end}}
   repository: registry.redhat.io/rhacm2
   version: v0.12.0
+  version: v0.12.1
```

Comment:
We are adding a second version. Is this valid or a typo?

Reply:
Oops, a mis-merge, thanks!